src/parser.h

```diff
@@ -146,8 +149,25 @@ struct profile_item_t {
 
 class parse_execution_context_t;
```

Comment:
not the highest priority but we should probably move the entire `library_data_t` to a `cxx::bridge` section.
Then we could share everything (but need to convert `wcstring` members to `std::unique_ptr` or `std::shared_ptr` etc)

Reply:
> we should probably move the entire library_data_t to a cxx::bridge section.

actually I think it's better if we make `LibraryData` a fully native Rust type and only expose getters/setters to C++. I'm working on that